templates/threatconnect.hbs

```diff
@@ -1000,31 +1063,161 @@
+                                (get newCaseAttributeValues case.id)
+                                case.selectedAttribute
+                              }}
+                              {{on 'change' (fn (action 'updateAttributes' case.id))}}
+                            >
+                              {{#each caseAttributeTypes.data as |attribute|}}
+                                <option value='{{json-stringify attribute}}'>
+                                  {{attribute.name}}
+                                </option>
+                              {{/each}}
+                            </select>
+                          </div>
+                        {{/if}}
                       </div>
+
+                      {{#if case.attributes.data.length}}
+                        <div class='info-bar-pill' style='width: 100%; padding-bottom: 5px;'>
+                          <span class='info-bar-key'>Case Attributes</span>
+                          {{#each case.attributes.data as |caseAttr|}}
+                            <div class='info-bar-value'>
+                              <div><strong>{{caseAttr.type}}</strong></div>
+                              <div>{{caseAttr.value}}</div>
+                            </div>
+                          {{/each}}
+                        </div>
+                      {{else}}
+                        <span class='info-bar-key'>No Attributes Associated</span>
+                      {{/if}}
+
+                      {{#if (get isEditingCases case.id)}}
+                        {{#if (get newCaseAttributes case.id)}}
+                          <div class='info-bar-pill' style='width: 100%; padding-bottom: 5px;'>
+                            <span class='info-bar-key'>New Case Attributes</span>
+                            {{#each (get newCaseAttributes case.id) as |caseAttr index|}}
+                              <div style="display: flex; justify-content: space-between;">
+                                <div class='info-bar-value'>
+                                  <div><strong>{{caseAttr.type}}</strong></div>
+                                  <div>{{caseAttr.value}}</div>
+                                </div>
+                                <button class="remove-attribute" {{on 'click' (fn (action 'removeAttribute' case.id index))}}>
+                                  {{fa-icon icon='times'}}
+                                </button>
+                              </div>
+                            {{/each}}
+                          </div>
+                        {{/if}}
+                      {{/if}}
+
+
+                      <button
+                        class='btn btn-secondary btn-add-tag'
+                        style='border-radius: 0px;'
+                        {{action 'toggleEdit' case.id orgData.indicator.id}}
+                      >
+                        {{if (get isEditingCases case.id) 'Save' 'Edit'}}
+                      </button>
```

Comment:
We need a cancel button here that reverts "toggleEdit" so you can get out of editing the case if you don't want to edit it.  Right now you can get "stuck" in this view:

![image](https://github.com/user-attachments/assets/60187065-a150-480c-bcd9-289113194b3f)